src/IHaskell/Flags.hs

```diff
@@ -95,6 +97,20 @@ help mode = showText (Wrap 100) $ helpText [] HelpFormatAll $ chooseMode mode
 ghcLibFlag :: Flag Args
 ghcLibFlag = flagReq ["ghclib", "l"] (store GhcLibDir) "<path>" "Library directory for GHC."
 
+ghcRTSFlag :: Flag Args
+ghcRTSFlag = flagReq ["use-rtsopts"] storeRTS "\"<flags>\""
+                  "Runtime options (multithreading etc.). See `ghc +RTS -?`."
+ where storeRTS allRTSFlags (Args mode prev)
+          = fmap (Args mode . (:prev) . RTSFlags)
+              . parseRTS . words $ filter (/='"') allRTSFlags
+       parseRTS ("+RTS":fs)  -- Ignore if this is included (we already wrap
+           = parseRTS fs     -- the ihaskell-kernel call in +RTS <flags> -RTS anyway)
```

Comment:
I suppose people might think the syntax is
```
ihaskell install --use-rtsopts="+RTS -N4 -RTS"
```
...but actually just
```
ihaskell install --use-rtsopts="-N4"
```
is needed. Rather than documenting this, it's probably easier to just strip the superfluous `±RTS` flags...